S3 Object Storage - Update Limitations

agoude opened the pull request:
new categories
added details on performances (bandwidth and rps)

agoude committed:
Update Limitations EN-GB

new categories
added details on performances (bandwidth and rps)

Y0Coss committed:
Proofreading EN version

Y0Coss committed:
FR version + EN minor updates

Y0Coss commented:
@agoude , please review the FR version. In the future, I would recommend updating the en-gb and fr-fr versions.

agoude commented on pages/storage_and_backup/object_storage/s3_limitations/guide.fr-fr.md:
```diff
@@ -1,29 +1,52 @@
+
+1 Gbps / connexion.
+
+Étant donné que la solution **S3 Object Storage OVHcloud** est un système hautement distribué, l’utilisation de **requêtes parallèles** vous aidera à surmonter cette limitation. En fonction de votre application et de votre cas d'usage, cette opération peut être effectuée en initiant simultanément des demandes (également appelées demandes simultanées).
```
peut-on juste remplacer 'commandes' par 'requêtes' stp? sinon, all good ;)

lason-ovh committed:
Update guide.en-gb.md

added recommandation for bucket names regarding issues with TLS certificate compatibility

lason-ovh committed:
Update guide.fr-fr.md

fr version

Commit:
Add duplications

Y0Coss committed:
Proofreading and date update

Y0Coss commented:
Thank you for your contribution @agoude
